### PR TITLE
when aggregate is being saved, events gets the same message ID as message transporting aggregate

### PR DESCRIPTION
**Pull request by unixslayer**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,6 +1,33 @@
 version: "3.8"
 
 services:
+  app8_2:
```
Comment (dgafka) [inline docker-compose.yml:4]: Can you bump up `app` service to php 8.2 instead?

File: quickstart-examples/EmittingEventsFromProjection/composer.json [hunk 2]
```diff
@@ -9,7 +9,11 @@
     ],
     "autoload": {
         "psr-4": {
-            "App\\": "src/"
+            "App\\": "src/",
+            "Ecotone\\": "../../packages/Ecotone/src",
+            "Ecotone\\EventSourcing\\": "../../packages/PdoEventSourcing/src",
+            "Ecotone\\JMSConverter\\": "../../packages/JmsConverter/src",
+            "Ecotone\\Lite\\": "../../packages/LiteApplication/src"
```
Comment (dgafka) [inline quickstart-examples/EmittingEventsFromProjection/composer.json:16]: Why this change? :thinking:

File: packages/Amqp/composer.json [hunk 3]
```diff
@@ -40,7 +40,7 @@
     "require": {
         "ext-amqp": "*",
         "ecotone/enqueue": "^1.82",
-        "enqueue/amqp-ext": "^0.10.4",
+        "enqueue/amqp-ext": "^0.10.18",
```
Comment (dgafka) [inline packages/Amqp/composer.json:43]: :+1:

File: quickstart-examples/EmittingEventsFromProjection/composer.json (hunk 2 above)
Comment (unixslayer) [reply quickstart-examples/EmittingEventsFromProjection/composer.json:16]: so `quickstart-examples` will run the current version of ecotone packages instead of latest one which is not the same. my changes from yesterday introduced a bug which was caught after releasing new version
